Dockerfile

```
@@ -0,0 +1,28 @@
+FROM koalaman/shellcheck-alpine as verify-sh
+WORKDIR /src
+COPY ./*.sh ./
+RUN shellcheck -e SC1091,SC1090 ./*.sh
+
+FROM node:14.17.4 AS restore
+WORKDIR /src
+COPY package.json yarn.lock ./
+RUN yarn
+COPY . .
+
+FROM restore AS verify-format
+RUN yarn verify-format
+
+FROM restore as build
+ARG baseUrl=https//cdn.fromdoppler.com
+ARG pkgName=unlayer-editor
+ARG version=v0.0.0-build0
+RUN yarn build
+
+FROM build AS test
+RUN yarn test:ci
```

Que opinás de setear aqui la variable de entorno para no requerir agregar una nueva entrada en el package.json?

```suggestion
FROM build AS test
ENV CI=true
RUN yarn test
```

---

Changed.